guide/how-it-works/stabilizing-bitcoin-value.md

```diff
@@ -0,0 +1,258 @@
+The stability only works for members of this platform
+
+{% include fact/close.html %}
+
+Examples are [Stablesats](https://stablesats.com), [Tau Protocol](https://www.tauprotocol.org), and [Standard Sats](https://standardsats.github.io/).
+
+## Discrete Log Contracts (DLCs)
+
+This technique is similar to synthetic currencies, but implemented in a different way via [Discreet Log Contracts for Difference](https://suredbits.com/settlement-of-dlcfd/) (we will call this smart contract from here on). The big change is that there's no need to trust an exchange. Anyone can be the counterparty, and all the rules that govern this special money are written into an immutable smart contract.
+
+There are three main entities in this. First, there's Alice who likes having a stable currency. Then, there's Robert (the counterparty) who is willing to enter into this financial contract with Alice (the motivation might be a bet that the price of bitcoin will go up). Finally, there's the oracle, whose job it is to provide the bitcoin to euro exchange rate to the contract, so it can enforce the rules.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/dlc-logic.png"
+   retina = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/dlc-logic@2x.png"
+   mobile = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/dlc-logic-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/dlc-logic-mobile@2x.png"
+   alt-text = ""
+   width = 1200
+   height = 400
+   layout = "full-width"
+%}
+
+So Alice enters an agreement with Robert via a smart contract, which automatically adjusts their respective amounts of bitcoin so she has a stable relative euro value.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/dlc-example.png"
+   retina = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/dlc-example@2x.png"
+   mobile = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/dlc-example-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/stabilizing-bitcoin-value/dlc-example-mobile@2x.png"
+   alt-text = ""
+   width = 1200
+   height = 400
+   layout = "full-width"
+%}
+
+In theory, this could even be extended to work within lightning channels. This has the advantages of being open-source and self-custodial, but comes at the cost of more technical complexity.
+
+{% include fact/pros.html %}
+
+- Option to have a stable asset
+- The stability is not scoped to a particular app platform, but instead can be implemented by many apps
+- Self-custodial and open-source
+
+{% include fact/close.html %}
+
+{% include fact/cons.html %}
+
+- The oracle goes down, so the smart contract no longer knows what the real-world price is. A default path in the DLC must be exercised and the user reclaims their original quantity of bitcoin but loses the price stability.
```

Comment:
As long as Robert and Alice cooperate they don't need the oracle. The oracle is only needed if the two do not cooperate.

Reply:
That's interesting. I assumed the process was fully automated based on the oracle data, but sounds like the oracle is there for conflict resolution.

So what's the process for Robert and Alice to cooperate on this? One of them requests a "closing" of the contract with a specific outcome, and the other has to agree/sign?

Reply:
> So what's the process for Robert and Alice to cooperate on this? One of them requests a "closing" of the contract with a specific outcome, and the other has to agree/sign?

Yes exactly. We call it _collaborative close_. In general it behaves similar to how lightning resolves HTLCs: 
`Look, I know the secret, I could go on-chain and get the money from the HTLC ... or, we could resolve this off-chain` vs. `Look, the oracle would say price is X, I could go on-chain and get the money ... or, we could resolve this off-chain`.
Obviously, if the oracle is not there anymore, a non-cooperative party could keep the money hostage or say you have to go on-chain... and if then there is still on oracle, a refund would come into play and both parties would get their money back.

Reply:
I updated the text (not where you left the comment, but a bit further up the page where oracles are first described). Thanks for the input.